exchanges/bitfinex/bitfinex.go

```diff
@@ -118,6 +119,142 @@ func (b *Bitfinex) GetPlatformStatus() (int, error) {
+	}
+	tempResp.UserPNL, ok = tempData[0].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for UserPNL", errTypeAssert)
+	}
+	tempResp.UserSwaps, ok = tempData[1].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for UserSwaps", errTypeAssert)
+	}
+	tempResp.MarginBalance, ok = tempData[2].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for MarginBalance", errTypeAssert)
+	}
+	tempResp.MarginNet, ok = tempData[3].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for MarginNet", errTypeAssert)
+	}
+	tempResp.MarginMin, ok = tempData[4].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for MarginMin", errTypeAssert)
+	}
+	return append([]MarginInfoV2{}, tempResp), nil
+}
+
+func symbolMarginInfo(data []interface{}) ([]MarginInfoV2, error) {
+	var resp []MarginInfoV2
+	for x := range data {
+		var tempResp MarginInfoV2
+		tempData, ok := data[x].([]interface{})
+		if !ok {
+			return nil, fmt.Errorf("%w for all sym", errTypeAssert)
+		}
+		var check bool
+		tempResp.Symbol, check = tempData[1].(string)
+		if !check {
+			return nil, fmt.Errorf("%w for symbol data", errTypeAssert)
+		}
+		tempFloatData, check := tempData[2].([]interface{})
+		if !check {
+			return nil, fmt.Errorf("%w for symbol data", errTypeAssert)
+		}
+		if len(tempFloatData) < 4 {
+			return nil, errors.New("invalid data received")
+		}
+		tempResp.TradableBalance, ok = tempFloatData[0].(float64)
+		if !ok {
+			return nil, fmt.Errorf("%w for TradableBalance", errTypeAssert)
+		}
+		tempResp.GrossBalance, ok = tempFloatData[1].(float64)
+		if !ok {
+			return nil, fmt.Errorf("%w for GrossBalance", errTypeAssert)
+		}
+		tempResp.BestAskAmount, ok = tempFloatData[2].(float64)
+		if !ok {
+			return nil, fmt.Errorf("%w for BestAskAmount", errTypeAssert)
+		}
+		tempResp.BestBidAmount, ok = tempFloatData[3].(float64)
+		if !ok {
+			return nil, fmt.Errorf("%w for BestBidAmount", errTypeAssert)
+		}
+		resp = append(resp, tempResp)
+	}
+	return resp, nil
+}
+
+func defaultMarginV2Info(data []interface{}) ([]MarginInfoV2, error) {
+	var tempResp MarginInfoV2
+	var ok bool
+	tempResp.Symbol, ok = data[1].(string)
+	if !ok {
+		return nil, fmt.Errorf("%w for symbol", errTypeAssert)
+	}
+	tempData, check := data[2].([]interface{})
+	if !check {
+		return nil, fmt.Errorf("%w for symbol data", errTypeAssert)
+	}
+	if len(tempData) < 4 {
+		return nil, errors.New("invalid data received")
+	}
+	tempResp.TradableBalance, ok = tempData[0].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for TradableBalance", errTypeAssert)
+	}
+	tempResp.GrossBalance, ok = tempData[1].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for GrossBalance", errTypeAssert)
+	}
+	tempResp.BestAskAmount, ok = tempData[2].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for BestAskAmount", errTypeAssert)
+	}
+	tempResp.BestBidAmount, ok = tempData[3].(float64)
+	if !ok {
+		return nil, fmt.Errorf("%w for BestBidAmount", errTypeAssert)
+	}
+	return append([]MarginInfoV2{}, tempResp), nil
```

Comment:
```suggestion
	return []MarginInfoV2{tempResp}, nil
```

However, this func and base margin info looks like it only returns a single data object despite the API returning an array so you could rid returning one too.